### PR TITLE
2022 12 06 external links

### PR DESCRIPTION
**Pull request by ghost**
**Related issues**

https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/19

### CODE REVIEW

**Comment by odscjames** [comment]
Only content change is fixing broken internal links - do you want a changelog entry for that?

**Comment by ghost** [comment]
This doesn't pass - there are 2 broken issues to do with anchors.

* GitHub one - I don't think that will ever pass. It looks like GH isn't using proper anchor tags and is instead using JS to scroll. The link checker will never pick that up.
* https://datatracker.ietf.org/doc/html/rfc7946#ref-WGS84 - this one seems odd - it looks like the source page is currently broken to. I can look in this more later.

Q: Do you want to review and maybe merge this now anyway? It would mean we can't set linkcheck as a required test before merging but I don't know if you want that anyway. It's really annoying to go to tweak a page and have your PR blocked due to an external change that is nothing to do with your edits. This would add a useful test that people can look in the details off.

**Comment by ghost** [comment]
Discussed Q and which items required a change entry with @lgs85 on call.

File: docs/guidance/publication.md [hunk 1]
```diff
@@ -110,29 +110,12 @@ To meet the widest range of use cases, you ought to publish data in all three fo
 ::::{tab-set}
 
 :::{tab-item} JSON to GeoJSON
-The standard repository's [`manage.py`](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/blob/main/manage.py) file provides a command-line interface for transforming OFDS data from JSON to GeoJSON format.
+[Lib CoVE OFDS](https://libcoveofds.readthedocs.io/en/latest/) provides a command-line interface for transforming OFDS data from JSON to GeoJSON format.
```
Comment (lgs85) [inline docs/guidance/publication.md]:
```suggestion
The command line tool [Lib CoVE OFDS](https://libcoveofds.readthedocs.io/en/latest/) provides an interface for transforming OFDS data from JSON to GeoJSON format.
```

File: docs/guidance/publication.md [hunk 2]
```diff
@@ -110,29 +110,12 @@ To meet the widest range of use cases, you ought to publish data in all three fo
...
-```bash
-pip install -r requirements.txt
-```
-
-- Run the following command:
+To convert a network package to GeoJSON format, follow the [installation instructions](https://github.com/Open-Telecoms-Data/lib-cove-ofds#installation) and then run the following command:
 
 ```bash
-./manage.py convert-to-geojson network-package.json
+libcoveofds jsontogeojson
```
Comment (lgs85) [inline docs/guidance/publication.md]:
```suggestion
libcoveofds jsontogeojson network-package.json nodes.geojson spans.geojson
```

File: docs/guidance/publication.md [hunk 3]
```diff
@@ -110,29 +110,12 @@ To meet the widest range of use cases, you ought to publish data in all three fo
...
-```
-
-- Install requirements
-
-```bash
-pip install -r requirements.txt
-```
-
-- Run the following command:
+To convert a network package to GeoJSON format, follow the [installation instructions](https://github.com/Open-Telecoms-Data/lib-cove-ofds#installation) and then run the following command:
```
Comment (lgs85) [inline docs/guidance/publication.md]:
```suggestion
To convert a network package to GeoJSON format, first install Lib CoVE OFDS by following the [installation instructions](https://github.com/Open-Telecoms-Data/lib-cove-ofds#installation). Then, run the following command:
```

**Review by lgs85 — CHANGES_REQUESTED** [review]
Looking good. Just a couple of things on usage of `libcodeofds` to check...

**Review by lgs85 — APPROVED** [review]

**Comment by lgs85** [comment]
for clarity, @odscjames and i discussed and agreed to merge this for now, and deal with the issues re failing link checks separately.

**Comment by duncandewhurst** [comment]
>  It would mean we can't set linkcheck as a required test before merging but I don't know if you want that anyway. It's really annoying to go to tweak a page and have your PR blocked due to an external change that is nothing to do with your edits. This would add a useful test that people can look in the details off.

That sounds fine to me - see my comment in https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/19#issuecomment-1223093957

**Pull request merged**